src/components/Reown/ReownModal.js

```diff
@@ -122,8 +135,10 @@ const baseStyles = StyleSheet.create({
 
 export const ReownModalTypes = {
   CONNECT: 'CONNECT',
```

Comment:
Does it make sense to change this type to `Connect` to match the others?

Reply:
I don't see why not